docs/source/optimization.rst

```diff
@@ -0,0 +1,140 @@
+.. module:: statsmodels.base.optimizer
+.. currentmodule:: statsmodels.base.optimizer
+
+Optimization
+============
+
+Many of the models we use such as :ref:`GLM <glm>` and
+:ref:`discrete models <discretemod>` allow for the optional selection of a
+scipy optimizer. This can either be defaut or an option based on the model
+selected.
+
+statsmodels supports the following optimizers along with keyword arguments
+associated with that specific optimizer:
+
+- ``newton`` - Newton-Raphson iteration. While not directly from scipy, we
+  consider it an optimizer because only the objective function and the score
+  are required parameters.
```

Comment:
newton also need the hessian

Reply:
Typo. Thanks. I meant to say it just needs score and hessian... It doesn't require objective function (loglike) which is super-helpful it turns out for Tweedie where loglike is difficult.